cob_cartesian_controller/src/trajectory_profile_generator/trajectory_profile_generator_ramp.cpp

```diff
@@ -0,0 +1,155 @@
+/*!
+ *****************************************************************
+ * \file
+ *
+ * \note
+ *   Copyright (c) 2015 \n
+ *   Fraunhofer Institute for Manufacturing Engineering
+ *   and Automation (IPA) \n\n
+ *
+ *****************************************************************
+ *
+ * \note
+ *   Project name: care-o-bot
+ * \note
+ *   ROS stack name: cob_control
+ * \note
+ *   ROS package name: cob_cartesian_controller
+ *
+ * \author
+ *   Author: Christoph Mark, email: christoph.mark@ipa.fraunhofer.de / christoph.mark@gmail.com
+ *
+ * \date Date of creation: September, 2015
+ *
+ * \brief
+ *
+ ****************************************************************/
+
+#include "ros/ros.h"
+#include "cob_cartesian_controller/trajectory_profile_generator/trajectory_profile_generator_ramp.h"
+/* BEGIN TrajectoryProfileRamp ********************************************************************************************/
+inline cob_cartesian_controller::ProfileTimings TrajectoryProfileRamp::getProfileTimings(double Se, double te, double accl, double vel, bool calcMaxTe)
+{
+    CartesianControllerUtils utils;
+    cob_cartesian_controller::ProfileTimings pt;
+    int steps_te, steps_tv, steps_tb = 0;
+    double tv, tb = 0.0;
+
+    // Calculate the Sinoid Profile Parameters
+    if (vel > sqrt(std::fabs(Se) * accl))
+    {
+        vel = sqrt(std::fabs(Se) * accl);
+    }
+
+    if(vel > 0.00001)
```

Comment:
please use a `#define`
is `0.00001` used as some kind of https://github.com/ipa-fxm/cob_control/blob/test_new_cartesian_controller/cob_twist_controller/include/cob_twist_controller/cob_twist_controller_data_types.h#L45-L46?